docs/reference/api/events/subscribe.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Subscribe to Events
+---
+
+Resources that can subscribe to a topic, such as [functions](/docs/intro/concepts/functions) and containers provide a `triggers` property defining their triggers. Adding a new `topics` object under `triggers`, followed by the name of the topic(s) to subscribe to will set up a subscription between the resource and the topic(s). In the case of functions, this causes the function to be invoked each time a new event is published to the topic. The function will receive a trigger with the event being provided as its payload. This is a useful mechanism when one service needs to communicate with another in order to have the second service perform some work.
+
+```yaml
+topics:
+	# A topic for events about newly registered users
+	user-registered: {}
+
+services:
+  # Registers new users and publishes an event
+  create-user:
+    handler: functions/user/register.ts
+  # Subscribes to all new user registrations and sends them a welcome email
+  welcome-user:
+    handler: functions/user/welcome.ts
+    triggers:
+      topics:
+        - user-registered
```

Comment:
I think this is a good example of duplicate content (a positive example that is), where the api section provides a call to action, but this same content will also appear under the function/container references in the project configuration above (as documented properties of those resources). But in this case we can just provide a link from the trigger property as well.